### PR TITLE
Added new Common Scripts

### PR DESCRIPTION
**Pull request by fvigo**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added 5 new Common Scripts (used in the Expanse Pack but generic enough to be reused)

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation 

Co-Author: @jtschichold

### CODE REVIEW

File: Packs/CommonScripts/Scripts/AddKeyToList/AddKeyToList_test.py [hunk 1]
```diff
@@ -0,0 +1,253 @@
+from AddKeyToList import add_key_to_list_command
```
Comment (esharf) [inline Packs/CommonScripts/Scripts/AddKeyToList/AddKeyToList_test.py:1]: unit tests should contain given-when-then comment to describe the unit test scenario.
\-Given
    \<some input>
\- When
    \<some state>
Relevant also for all the unit tests files.
\- Then
    \<ensure some behviour>

**Review by esharf — CHANGES_REQUESTED** [review]

**Comment by fvigo** [comment]
@esharf thanks for the review. changes applied, please recheck.

**Review by esharf — APPROVED** [review]
@fvigo please fix the conflict and we are ready to merge.

**Comment by fvigo** [comment]
@esharf done

**Pull request merged**